src/topology/metric_space/emetric_space.lean

```diff
@@ -495,13 +495,15 @@ def ball (x : α) (ε : ℝ≥0∞) : set α := {y | edist y x < ε}
 
 @[simp] theorem mem_ball : y ∈ ball x ε ↔ edist y x < ε := iff.rfl
 
-theorem mem_ball' : y ∈ ball x ε ↔ edist x y < ε := by rw edist_comm; refl
+theorem mem_ball' : y ∈ ball x ε ↔ edist x y < ε := by {rw edist_comm, refl}
```

Comment:
```suggestion
theorem mem_ball' : y ∈ ball x ε ↔ edist x y < ε := by rw [edist_comm, mem_ball]
```
(untested)

Reply:
Generally speaking, there should always be spaces inside `by {}`

Reply:
Have changed these, I think. Forgot about edist.